Initial step for PBS hitboxes fix

Sellafield opened the pull request:
- New commands ZonePBSApplyStagingToConstructed and ZonePBSReplaceActualWithStaging
- Fixed encoding issues in few files
- Minor style guide applied

Ghroom committed:
Logging - Chat Commands (#315)

Ghroom committed:
added dipslay of the command if it's an admin

Commit:
Initial step for PBS hitboxes fix

- New commands ZonePBSApplyStagingToConstructed and ZonePBSReplaceActualWithStaging
- Fixed encoding issues in few files
- Minor style guide applied

Commit:
- Affected buildings limited to just Repair node, Mining outpost, Energy well and Command relay

Commit:
- Removing files with wrong encoding

VisualTunic488 commented:
Notes on the progress of the review for today can be found here https://discord.com/channels/340853262288814080/389468976436215808/1146265715138101309

VisualTunic488 commented:
----Live Patch_29 Testing round 2
--CTs for Lights and Assaults buff for artefacts on gamma 
#Testing in progress

Tests Success, no changes needed--Scarab Changes  (testing done with maxed stats)
#Deploying works fine when reactor is filled to the new max of 1050.
#Deploying works fine when CPU is filled to the new max of 560.

Tests Success, no changes needed--Scarab mk2 Changes  (testing done with maxed stats)
#Deploying works fine for both reactor being maxed to 1134 and CPU being maxed to 616
#New leg and head slots have been applied

--Hit Box for PBS Structures 
#Change Required, The following structures were tested:
	#-Miner tower
	#Unable to be damaged still on test server
	#-Energy field tower
	#Unable to be damaged still on test server
	#-Command relay
	#Unable to be damaged still on test server

--Chat Logging command 
#Need clarification, what these commands are/what they do

--Starter Artefact Change 
#Change required, https://discord.com/channels/340853262288814080/389468976436215808/1146239577816846387

Tests complete, no changes needed--Metis Leg Slot +1 and Speed increase  (testing done with maxed stats)
#Test Success, slot is available and functional

VisualTunic488 commented:
--Hit Box for PBS Structures
#Disregard the last post about this. They suddenly work. Trying to identify why they didnt before

VisualTunic488 commented:
Update on --Hit Box for PBS Structures
Reason it didnt work before was because that had to be applied to all islands individually. when done it worked fine
![image](https://github.com/OpenPerpetuum/PerpetuumServer/assets/32330431/7c8c2e47-70b9-4a3a-b1a8-cafe7aa33d2f)

Commit:
CommandLog

- Style-guide fixes. also removed unused variables

Commit:
CommandLog

- Removing files with wrong encoding

Commit:
CommandLog

- Re-adding files with correct encoding

Commit:
CommandLog

- minor script change

Commit:
PBS Hitboxes Fix

- DevMode is not required to run commands anymore
- ZonePBSApplyStagingToConstructed command is now server-wide and will apply to each gamma
